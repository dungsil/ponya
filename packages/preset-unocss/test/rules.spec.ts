import { describe, test } from 'vitest'
import { matches } from './utiles'


describe('Fonts', () => {
  describe('font family', () => {
    test('font-sans', async () => await matches(['font-sans']))
  })

  describe('font-size', () => {
    test('text-ui', async () => await matches(['text-ui']))
    test('text-base', async () => await matches(['text-base']))
    test('text-title', async () => await matches(['text-title']))
    test('text-sub', async () => await matches(['text-sub']))
  })
})
