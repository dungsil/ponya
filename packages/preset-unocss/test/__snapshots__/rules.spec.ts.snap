// Vitest Snapshot v1

exports[`Fonts > font family > font-sans 1`] = `
"/* layer: default */
.font-sans{font-family:Pretendard, -apple-system, BlinkMacSystemFont, system-ui, Roboto, \\"Helvetica Neue\\", \\"Segoe UI\\", \\"Apple SD Gothic Neo\\", \\"Noto Sans KR\\", \\"Malgun Gothic\\", \\"Apple Color Emoji\\", \\"Segoe UI Emoji\\", \\"Segoe UI Symbol\\", sans-serif;}"
`;

exports[`Fonts > font-size > text-base 1`] = `
"/* layer: default */
.text-base{font-size:1.125rem;line-height:0.05555rem;}"
`;

exports[`Fonts > font-size > text-sub 1`] = `
"/* layer: default */
.text-sub{font-size:0.875rem;line-height:1;}"
`;

exports[`Fonts > font-size > text-title 1`] = `
"/* layer: default */
.text-title{font-size:2.25rm;line-height:2.5rem;}"
`;

exports[`Fonts > font-size > text-ui 1`] = `
"/* layer: default */
.text-ui{font-size:1rem;line-height:0;}"
`;
